NPS.ID.PublicApi.DotNet.Client/Connection/Clients/StompClient.cs

```diff
@@ -0,0 +1,223 @@
+using System.Text;
+using System.Threading.Channels;
+using Microsoft.Extensions.Logging;
+using Microsoft.VisualStudio.Threading;
+using Newtonsoft.Json;
```

Comment:
We should use System.Text.Json if possible, it's a recommended choice for new apps

Reply:
I am trying to switch into System.Text.Json and I am facing with an error while OrderExecutionReport deserialization. 
OrderExecutionReport object has not nullable UpdatedAt field, but we can receive as response json with null in this field.
Basically Newtonsoft.Json was able to handle it and it did set default value for UpdatedAt field while System.Text.Json throws an exception in such situation:

> System.Text.Json.JsonException: The JSON value could not be converted to System.DateTimeOffset. Path: $[0].updatedAt | LineNumber: 0 | BytePositionInLine: 18.
>  ---> System.InvalidOperationException: Cannot get the value of a token type 'Null' as a string.

Reply:
So I wonder if we should stay with Newtonsoft.Json, update public api OrderExecutionReport model and set UpdatedAt field as nullable or is there any other way to overcome this deserialization error.

Reply:
For now in this single place I decided to stay with Newtonsoft.Json as there is no simply way to handle nullable values for non nullable value types like integer or DateTime. 

Possible options to handle this issue are:
- creating custom converter for each type, or single converter that will handle all valuable types (this issue occurs also on EventSequenceNo property of OrderExecutionEntry model)
- update models and change properties as nullable, the thing is that, for UpdatedAt it will have impact on all other models as it is part of parent type inherited by other models, we would have to get rid of this property from BaseRow and add it for each model type
- there is [opened issue](https://github.com/dotnet/runtime/issues/66490) with change on to handle such cases

Reply:
Adjusted models to be valid for OER response (some fields changed as nullable), completely removed Newtonsoft.Json dependency